### PR TITLE
added two new methods to QuantityUtil

### PR DESCRIPTION
**Pull request by johanneshiry**
transfer of the until now internal `QuantityUtil.considerablyAbsEqual(...)` and `QuantityUtil.considerablyRelEqual(...)` to its correct util class

### CODE REVIEW

**Comment by johanneshiry** [comment]
!test

**Comment by johanneshiry** [comment]
!test

**Comment by johanneshiry** [comment]
!test

File: src/main/java/edu/ie3/util/quantities/QuantityUtil.java [hunk 1]
```diff
@@ -25,4 +25,51 @@ private QuantityUtil() {
...
+   * @param b Second quantity to compare
+   * @param absQuantityTolerance Permissible absolute tolerance
+   * @param <Q> Type of Quantity
+   * @return true, if both quantities' values differ less then the given tolerance else false
+   */
+  public static <Q extends Quantity<Q>> boolean considerablyAbsEqual(
+      Quantity<Q> a, Quantity<Q> b, double absQuantityTolerance) {
+
+    // if units differ, return false
+    if (!a.getUnit().equals(b.getUnit())) return false;
```
Comment (ckittl) [inline src/main/java/edu/ie3/util/quantities/QuantityUtil.java:44]: My initial intention when designing the internal method was to compare two things, that are semantically equal. Therefore, my method didn't have the check for equal units. I will leave it up to you, if you either want to provide an additional method, that is providing this functionality or just remove this line and line 68. BTW: Unit conversion in lines 47 and 71 are pointless, as this line ensures, that the units are already the same.

Comment (ckittl) [reply src/main/java/edu/ie3/util/quantities/QuantityUtil.java:44]: BTW: Added a test case to the test class.

**Pull request merged**